CHANGELOG.md

```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2020.06.0
+
+### Added
+- Algorithms
+  * PPO in PyTorch (
+    [#905](https://github.com/rlworkgroup/garage/pull/905),
+    [#1188](https://github.com/rlworkgroup/garage/pull/1188))
+  * TRPO in PyTorch (
+    [#1018](https://github.com/rlworkgroup/garage/pull/1018),
+    [#1053](https://github.com/rlworkgroup/garage/pull/1053))
+  * MAML in PyTorch ([#1136](https://github.com/rlworkgroup/garage/pull/1136),
+    [#1234](https://github.com/rlworkgroup/garage/pull/1234),
+    [#1283](https://github.com/rlworkgroup/garage/pull/1283))
+  * RL2 in TensorFlow
+    ([#1127](https://github.com/rlworkgroup/garage/pull/1127),
+    [#1190](https://github.com/rlworkgroup/garage/pull/1190),
+    [#1195](https://github.com/rlworkgroup/garage/pull/1195),
+    [#1231](https://github.com/rlworkgroup/garage/pull/1231))
+  * PEARL in PyTorch ([#1059](https://github.com/rlworkgroup/garage/pull/1059),
+    [#1124](https://github.com/rlworkgroup/garage/pull/1124),
+    [#1218](https://github.com/rlworkgroup/garage/pull/1218),
+    [#1316](https://github.com/rlworkgroup/garage/pull/1316),
+    [#1374](https://github.com/rlworkgroup/garage/pull/1374))
+  * SAC in PyTorch ([#1235](https://github.com/rlworkgroup/garage/pull/1235))
+  * MTSAC in PyTorch ([#1332](https://github.com/rlworkgroup/garage/pull/1332))
+  * Task Embeddings in TensorFlow
+    Add TensorFlow embedding models ([#1168](https://github.com/rlworkgroup/garage/pull/1168))
```

Comment:
i think rendering fails here?

Reply:
Oh, I just forgot to update that bit. Fixed.